### DISCUSSION TITLE
Discord Bot Rich Presence through EssentialsDiscord API?

### DISCUSSION THREAD

**Discussion by Somecode14** [EssentialsX/Essentials#5813] [Q&A] [answered]
**Hello!**
I am a developer of a Minecraft minigame event server. I'm wondering if it is possible to show the amount of online players, time elapsed and other info in the bot profile through [Discord Rich Presence](https://discord.com/developers/docs/rich-presence/how-to) or whatever it is called now, using EssentialsDiscord Developer API (if it is possible at all).

I am also not sure how can I update the bot status itself (like `Playing Minecraft`) without modifying the config file.

If those features are not available, it would be great to see them as a part of the API!

Thank you :D

**Comment by JRoy** [comment, marked as answer]
Bots are not allowed to display rich presence
